exercises/04-integration/after/Divergent.Customers/ProgramService.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.ComponentModel;
+using System.ServiceProcess;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+using Divergent.Customers.Config;
+
+namespace Divergent.Customers
+{
+    [DesignerCategory("Code")]
```

Comment:
What is this?

Reply:
It comes from the template, I can clean all this if we need to